### ISSUE TITLE
delete branch gh-pages

### ISSUE THREAD

**Issue by montoyamoraga** [processing/p5.js-sound#486]
to make everything easier to mantain, i would delete the branch gh-pages, and on settings, make the hosting be built from the master branch.

i don't know if something would break, or if any other file needs to be adapted for this to happen.

**Comment by therewasaguy** [comment]
Good idea. I want to figure out https://github.com/processing/p5.js-sound/issues/485 first
